src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -828,6 +829,80 @@ public static BinaryOperandsKind GetBinaryOperandsKind(BinaryOperationKind kind)
         }
     }
 
+    public interface IConversion
```

Comment:
If we do that, then we will still have two different ways of talking about conversions.  We'll have the SemanticModel/Compilation APIs that talk about conversions, and which talk about them differently than how IOperation talks about it.  This means on the IDE side we have to duplicate logic around conversions depending on how we obtained it from the compiler APIs.  This seems very ungainly and works against our goals of common, composable, APIs across VB and C#.  

I'd really like some design feedback on if this is the way we want to go.